### DISCUSSION TITLE
RESOLVED - reset mythicv2 command contains a bug

### DISCUSSION THREAD

**Discussion by matteo-merzi** [jon-heard/obsidian-inline-scripts-library#4] [Bugs and issues]
The `^reset mythic(?:v2)?$` command contains a bug. At line 86 it refers to the wrong state `mythicgme` instead of `mythicv2`.
```js
return "***\n\n\n### SCENE " + window._tejs.state.mythicv2.scene + "\n- Setup:\n    - ";
```
Above the corrected line.
I tried to push a new branch and open a PR but I do not have permission, so I am adding it here.
I hope you will see it 😜

**Comment by jon-heard** [comment]
Thank you so much for reporting this!  I've fixed the issue.
I __do__ check this discussion regularly.  I opened it in case people wanted to report bugs (thanks again), suggestions for improvements, or anything else library related.
